docs/spdx-tag-value-mapping.md

```diff
@@ -0,0 +1,151 @@
+SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Nice! Although this will get rendered. Can you try encapsulating this in `<!-- -->`
<!--SPDX-License-Identifier: BSD-2-Clause-->

Reply:
I tried it here in comments and it works!